lib/buildConnectorsJson/index.js

```diff
@@ -94,9 +94,13 @@ module.exports = function (directory, config) {
 
 	});
 
+	var jsonSchemaStringified = JSON.stringify(connectorsJson, null, '\t');
```

Comment:
well, you could return it as js object, without stringify, as connector builder needs to parse it anyway

Reply:
Ah, yh, I was wondering how you were processing it. If JSON object is fine instead of stringified, that would also make tests easier. 

I'll change that then.

Reply:
cool